src/main/webapp/js/feedbackResponseComments.js

```diff
@@ -66,8 +66,8 @@ var addCommentHandler = function(e) {
 var editCommentHandler = function(e) {
     var submitButton = $(this);
     var cancelButton = $(this).next("input[value='Cancel']");
-    var formObject = $(this).parent().parent();
-    var displayedText = $(this).parent().parent().prev();
+    var formObject = $(this).closest('form');
+    var displayedText = formObject.prev();
```

Comment:
Same here, if we can, that'll be better and it helps us be more flexible in future HTML changes